stable/prometheus-mongodb-exporter/values.yaml

```diff
@@ -49,6 +49,9 @@ resources: {}
 #   cpu: 100mm
 #   memory: 128Mi
 
+# Extra environment variables that will be passed into the exporter pod
+env: {}
```

Comment:
can you add a test case to test this?

Reply:
No problem, I've added one here https://github.com/helm/charts/pull/12615/commits/60b41f078f773ff7f281c82cf668185e4e72160e. I haven't added test for charts before so let me know if it's alright. Looks ok according to the docs anyway :slightly_smiling_face: